snapcraft/internal/sources.py

```diff
@@ -201,6 +201,27 @@ def pull(self):
         subprocess.check_call(cmd)
 
 
+class Subversion(Base):
+
+    def __init__(self, source, source_dir, source_tag=None,
+                 source_branch=None):
+        super().__init__(source, source_dir, source_tag, source_branch)
+        if source_tag:
+            raise IncompatibleOptionsError(
+                "can\'t specify a source-tag for a subversion source")
```

Comment:
@elopio Bazaar has the same problem, I'll correct Subversion, but might it be worth a whole other PR to fix this?